src/installing-guacamole.md

```diff
@@ -460,6 +460,13 @@ fail outright, and you will need to install the missing dependency. If, after
 running `configure`, you find support for something you wanted is missing,
 simply install the corresponding dependencies and run `configure` again.
 
+During the configure process, the script will attempt to detect the version of
+FreeRDP installed on your system. It will first look for FreeRDP version 3,
+indicated as freerdp3 ............ yes. If FreeRDP version 3 is not found, it will
+then check for FreeRDP version 2, which would be shown as
+freerdp2 ............ yes. The output will reflect the specific version found,
```

Comment:
It seems like the `freerdp3 ...... yes` and `freerdp2 ...... yes` items on these lines should use some formatting to indicate screen output?